gridappsd-field-bus-lib/gridappsd/field_interface/agents/agents.py

```diff
@@ -109,6 +113,13 @@ def connect(self):
         if ('context_manager' not in self.app_id):
             LocalContext.register_agent(self.downstream_message_bus,
                                         self.upstream_message_bus, self)
+            
+    def disconnect(self):
```

Comment:
But we aren't using _ here?

Reply:
yeah, agent uses intialization to connect but can disconnect is some properties in topology is not applicable to their app like not having any addressable equipment. Having initialization and connect function and spawn function was causing more confusion before.